src/Data/Functor/Extend.hs

```diff
@@ -80,6 +83,24 @@ class Functor w => Extend w where
   {-# MINIMAL duplicated | extended #-}
 #endif
 
+-- | Generic 'duplicated'. Caveats:
+--
+--   1. Will not compile if @w@ is a product type.
+--   2. Will not compile if @w@ is a recursive type.
+--
+-- @since 5.3.8
+gduplicated :: (Extend (Rep1 w), Generic1 w) => w a -> w (w a)
+gduplicated = to1 . fmap to1 . duplicated . from1
+
+-- | Generic 'extended'. Caveats:
+--
+--   1. Will not compile if @w@ is a product type.
+--   2. Will not compile if @w@ is a recursive type.
```

Comment:
Similarly, I can't think of a way to give `(:.:)` an `Extend` instance, so we should just change the wording here a bit.